lib/fluent/plugin/out_prometheus_pushgateway.rb

```diff
@@ -17,6 +17,14 @@
 require 'prometheus/client/push'
 require 'fluent/plugin/output'
 
+begin
+  require 'fluent/tls'
+rescue LoadError
+  # compatible layer for fluentd v1.9.1 or earlier
+  # https://github.com/fluent/fluentd/pull/2802
+  require 'fluent/plugin/tls'
```

Comment:
should be `require_relative`?

Reply:
Why do you think so? An absolute path is fine for me.

Reply:
What happens if other plugin have `fluent/plugin/tls` ?

Reply:
I created own directory for prometheus_pushgateway then.